src/libs/Violations/ViolationsUtils.ts

```diff
@@ -100,15 +126,41 @@ function getTagViolationsForMultiLevelTags(
                 break;
             }
         }
+
         if (!hasInvalidTag) {
             newTransactionViolations = reject(newTransactionViolations, {
                 name: CONST.VIOLATIONS.TAG_OUT_OF_POLICY,
             });
         }
     }
+
     return newTransactionViolations;
 }
 
+/**
+ * Calculates some tag levels required and missing tag violations for the given transaction
```

Comment:
It's more than just those 2, right?
```suggestion
 * Calculates tag violations for a transaction on a policy with multi level tags
```

Reply:
@cead22 affirmative, thanks for pointing it out 😄